src/styles/globalStyles.js

```diff
@@ -0,0 +1,40 @@
+import { createGlobalStyle } from 'styled-components';
+import reset from './reset';
+import fonts from './fonts';
+import theme from './theme';
+
+export default createGlobalStyle`
+    ${reset};
+    ${fonts};
+    
+    *,
+    *::before,
+    *::after{
+        margin:0;
+        padding:0;
+        box-sizing:inherit;
+    }
+
+    html{
+        box-sizing:border-box;
+        min-height:100%;
+        width:100%;
+    }
+    #root,
+    body{
+        width:100%;
+        height:100%;
+    }
+    body{
+        line-height:1.5;
+        background-color:${theme.color.grayLight};
```

Comment:
Can't you access the theme from the props like you usually would do? 

https://spectrum.chat/styled-components/general/i-cant-use-my-theme-in-createglobalstyle-function-styled-components-v4-react-v16-6-3~0978b404-ab71-45c9-8f75-0862abde4eb5

Reply:
I didn't know this, Thanks